src/pages/dmt5/_why-trader.js

```diff
@@ -82,58 +109,21 @@ const WhyTrader = () => {
                 {localize('Why trade with DMT5')}
             </StyledHeader>
             <CardContainer>
-                <Card>
-                    <div>
-                        <img src={InstantAccess} alt="instant access" />
-                    </div>
-                    <StyledCardHeader mt="0.8rem" mb="0.8rem" as="h4" type="sub-section-title">
-                        {localize('Quick demo account sign-up')}
-                    </StyledCardHeader>
-                    <StyledText>
-                        {localize(
-                            'Practise with a demo account preloaded with unlimited virtual funds.',
-                        )}
-                    </StyledText>
-                </Card>
-                <Card>
-                    <div>
-                        <img src={SyntheticIndices} alt="syntethic indices" />
-                    </div>
-                    <StyledCardHeader mt="0.8rem" mb="0.8rem" as="h4" type="sub-section-title">
-                        {localize('Multiple assets on a single platform')}
-                    </StyledCardHeader>
-                    <StyledText>
-                        {localize(
-                            'Explore forex, synthetic indices, stocks, and commodities on an all-in-one platform.',
-                        )}
-                    </StyledText>
-                </Card>
-                <Card>
-                    <div>
-                        <img src={Seven} alt="seven" />
-                    </div>
-                    <StyledCardHeader mt="0.8rem" mb="0.8rem" as="h4" type="sub-section-title">
-                        {localize('24/7 trading')}
-                    </StyledCardHeader>
-                    <StyledText>
-                        {localize(
-                            'Trade round-the-clock, even on weekends, with our proprietary synthetic indices.',
-                        )}
-                    </StyledText>
-                </Card>
-                <Card>
-                    <div>
-                        <img src={MinimalRisk} alt="minimal risk" />
-                    </div>
-                    <StyledCardHeader mt="0.8rem" mb="0.8rem" as="h4" type="sub-section-title">
-                        {localize('Licensed and regulated')}
-                    </StyledCardHeader>
-                    <StyledText>
-                        {localize(
-                            'Trade with a regulated industry pioneer trusted by traders for more than 20 years.',
-                        )}
-                    </StyledText>
-                </Card>
+                {cardContent.map((card, index) => {
+                    return (
+                        <Card key={index}>
```

Comment:
```suggestion
                        <Card key={card.alt}>
```
Best practice not to use `index` because of: https://programmingwithmosh.com/react/why-do-i-need-keys-in-react-lists/